src/Plugin.php

```diff
@@ -15,21 +15,21 @@ class Plugin extends BasePlugin
      *
```

Comment:
This class needs to be renamed to `WebservicePlugin`.

Reply:
Okay, changed it, but why? Doesn't this class always need to be called Plugin?

Reply:
Cake's 5 convention is that the plugins classes should be named as `{PluginName}Plugin`.

Reply:
Okay, didn't know that. Thanks for the heads up!